masterlist.yaml

```
@@ -14611,13 +14616,94 @@ plugins:
 
   - name: 'RUSTIC SOULGEMS - (Sorted|Unsorted)( - ESL)?( - GIST)?\.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/5785/' ]
+    tag: [ Graphics, ObjectBounds ]
```

If there are multiple items in a list then it should be expanded.

```suggestion
    tag:
      - Graphics
      - ObjectBounds
```